### DISCUSSION TITLE
How to use CMI Api

### DISCUSSION THREAD

**Discussion by Bloodlex** [Zrips/CMI#6222] [General]
Hey,

Could you please help me using CMI Api in my own plugin? I would like to get a list of vanished players, but I don't know how I can implement it in my own plugin to get that from CMI. There is no maven repository to add and I'm struggling with adding it localy, since I use maven to compile everything. Could you please help? Thanks!

**Comment by crazymarty** [comment]
If you are using IntelliJ you can simply add jar files using these steps:
• Click on File
• Click on Project Structure
• Click on Modules
• In your project click on Dependencies
• Hit "+" sign and click on JARs or Directories

Edit 2: (Sorry for multiple edits, there was a typo in first edit)
To get a list of vanished players, you simply use VanishManager.getAllVanished();
You can look here:
https://github.com/Zrips/CMI-API/blob/master/src/com/Zrips/CMI/Modules/Vanish/VanishManager.java

**Reply by Bloodlex** [reply]
Okay, thank you very much 🙂 I think it would be easier if you could provide maven artifact with CMI API, maybe you consider this 🙂
